src/main/java/net/fabricmc/loom/providers/MinecraftMappedProvider.java

```diff
@@ -118,8 +122,15 @@ private void mapMinecraftJar() throws IOException {
 	}
 
 	public TinyRemapper getTinyRemapper(String fromM, String toM) throws IOException {
+		TinyTree annotationMappingsTree;
+
+		try (BufferedReader reader = new BufferedReader(new InputStreamReader(MinecraftMappedProvider.class.getClassLoader().getResourceAsStream("jsrToJetbrains.tiny")))) {
```

Comment:
I would have done 
```
Map<String,String> jsrToJetbrains = new HashMap {{
   put("javax/annotation/Nullable", "org/jetbrains/annotations/Nullable");
   ...
}};
```
And then
```
.withMappings(acceptor -> jsrToJetbrains.forEach((jetbrains,jsr) -> acceptor.acceptClass(jetbrains,jsr)))
```

Reply:
Yes, thats prob better. I expected there to be more to remap possibly including methods.

Reply:
![Every time you use double brace initialization, god kills a kitten](https://2.bp.blogspot.com/-Ncf3U1GomEk/VhvGI50SkaI/AAAAAAAAD8w/sF5iX3k-66E/s400/Double%2BBrace%2BIniitialization%2Bin%2BJava%2BAntitpatter.jpg)